app/models/concerns/course_user/staff_concern.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+# This concern groups methods related to calculate staff performance.
+module CourseUser::StaffConcern
+  extend ActiveSupport::Concern
+
+  included do
+    # Sort the staff by their average marking time.
+    # Note that nil time will be considered as the smallest.
```

Comment:
Actually for the staff monitoring the first record should be the smallest, non-nil record. If nil is the smallest, we have phantom staff being at the top of this staff monitoring. 

Unless I understood this wrongly?